.molecule/default/setup_aiida.yml

```diff
@@ -36,7 +36,7 @@
     command: >
       {{ venv_bin }}/verdi setup
       --non-interactive
-      --profile "{{ aiida_backend }}"
+      --profile-name "{{ aiida_backend }}"
```

Comment:
this is not correct is it? This is using `verdi setup` which is not being affected.

Reply:
Good catch!